package.json

```diff
@@ -2864,6 +2864,144 @@
 												]
 											}
 										]
+									},
+									{
+										"key": "languageId:python",
+										"name": "Python",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format document",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "m",
+														"name": "Format modified lines only",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													},
+													{
+														"key": "S",
+														"name": "+Format selection with",
+														"type": "command",
+														"command": "editor.action.formatSelection.multiple"
+													}
```

Comment:
What does it do? For me it's not working

Reply:
I figured it out and commented it on https://github.com/VSpaceCode/VSpaceCode/issues/160#issuecomment-759887333. Let's continue the conversation on that issue.